mcs/class/corlib/corert/RuntimeThread.cs

```diff
@@ -33,5 +33,11 @@ public static bool SpinWait (int iterations)
 			Thread.SpinWait (iterations);
 			return true;
 		}
+
+		public static int GetCurrentProcessorId ()
+		{
+			// TODO: Implement correctly
+			return 1;
```

Comment:
should we use `Environment.CurrentManagedThreadId` instead? That's what they're doing in some cases in CoreCLR: https://github.com/dotnet/coreclr/blob/29d5dc25ab6fd078cc349e7f7e21aaeacda59acc/src/mscorlib/src/Internal/Runtime/Augments/RuntimeThread.cs#L234-L238